types/node/net.d.ts

```diff
@@ -38,22 +38,19 @@ declare module "net" {
         constructor(options?: SocketConstructorOpts);
 
         // Extended base methods
-        write(buffer: Buffer): boolean;
-        write(buffer: Buffer, cb?: Function): boolean;
-        write(str: string, cb?: Function): boolean;
-        write(str: string, encoding?: string, cb?: Function): boolean;
+        write(buffer: Buffer | Uint8Array | string, cb?: (err?: Error) => void): boolean;
+        write(str: string, encoding?: string, cb?: (err?: Error) => void): boolean;
         write(str: string, encoding?: string, fd?: string): boolean;
```

Comment:
I can't find the variant wiht `fd?: string` as third argument in docu. is this really existing?

Reply:
Removed.